keyvault/client/keys/src/main/java/com/azure/keyvault/keys/models/DeletedKey.java

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.keyvault.keys.models;
```

Comment:
Is it a standard in the java sdk that model types are in a model namespace?  This isn't the case in other languages

Reply:
It is one of the best practices. But arch board hasn't made any strict guidelines for it yet.
CC: @JonathanGiles